### PR TITLE
ERC20Locker: make AdminControlled. Add tests.

### PR DESCRIPTION
**Pull request by sept-en**
* Add `AdminControlled` contract.
* ERC20Locker: inherit from `AdminControlled`.
* ERC20Locker: make `lockToken` and `unlockToken` pausable.
* Add tests.

### CODE REVIEW

File: erc20-connector/test/TokenLocker.js
Comment (mfornet) [inline erc20-connector/test/TokenLocker.js:86]:
```suggestion
            const tx1 = await this.locker.lockToken(this.token.address, amountToTransfer, 'receiver', { from: addr1 });
```

File: erc20-connector/test/TokenLocker.js
Comment (mfornet) [inline erc20-connector/test/TokenLocker.js:88]:
```suggestion
            expect(fromWei(afterBalance1)).equal((fromWei(initialBalanceAddr1) - fromWei(amountToTransfer)).toString());
```

**Review by mfornet — APPROVED** [review]

File: erc20-connector/test/TokenLocker.js
Comment (sept-en) [reply erc20-connector/test/TokenLocker.js:86]: oops, copy-pasted this part from the old test.

**Review by mfornet — CHANGES_REQUESTED** [review]
This PR (https://github.com/near/rainbow-bridge/pull/496) is closed to be merged, only missing review but tests are already passing. Notice that `AdminControlled` interface was changed a little bit (all method can be called by Admin, either if them are paused or not). So I think we should merge this PR after that one is merged, and use `AdminControlled` as a dependency.

I will mark as request change, to avoid merging it for now!

File: erc20-connector/contracts/ERC20Locker.sol
Comment (djsatok) [inline erc20-connector/contracts/ERC20Locker.sol:33]:
```suggestion
    uint constant PAUSED_LOCK = 1 << 0;
    uint constant PAUSED_UNLOCK = 1 << 1;
```

File: erc20-connector/test/TokenLocker.js
Comment (djsatok) [inline erc20-connector/test/TokenLocker.js:214]: Check indents

**Review by djsatok — CHANGES_REQUESTED** [review]

**Comment by mfornet** [comment]
@sept-en https://github.com/near/rainbow-bridge/pull/496 is already merged. We can use Admin Controller as dependency now!

File: erc20-connector/test/TokenLocker.js
Comment (sept-en) [reply erc20-connector/test/TokenLocker.js:214]: These are ok and correspond to the file's indents.

**Comment by djsatok** [comment]
LGTM

**Comment by sept-en** [comment]
> Why tests are not triggered in buildkite, only building the contracts? @sept-en @mfornet

Good point. I added both ETH and NEAR side tests to the CI.

However, now I notice that one of the tests for `bridge-token-factory` - `test_near_token_transfer` fails with the following error: `Action #0: Smart contract panicked: panicked at 'Native NEP21 on Ethereum is disabled.', src/lib.rs:281:9) }'`

I believe that this test will not pass until the #39 is merged. So I temporarily disabled this test.

**Comment by mfornet** [comment]
> I believe that this test will not pass until the #39 is merged. So I temporarily disabled this test.

Actually we removed entirely the functionality for bridging native NEAR tokens to ethereum for now.

**Pull request merged**
